common/oatbox/session/SessionContext.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2020 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
+ *
+ */
+
+declare(strict_types=1);
+
+namespace oat\oatbox\session;
+
+/**
+ * Represents an additional context that can be attached to a session
+ */
+interface SessionContext
```

Comment:
Shouldn't this interface have some behavior defined? 

As a context it should at least have some contextData (_getContextData(): array_ and _addContextData(string $id, $data): void_) otherwise this interface will force user to break `open/closed` principle as the client of this context will have to be aware of implementation instead or relying in the interface.